src/manifest/dbtProject.ts

```diff
@@ -639,10 +640,20 @@ select * from renamed
   }
 
   private findTargetPath(projectConfig: any): string {
-    if (projectConfig[DBTProject.TARGET_PATH_VAR] !== undefined) {
-      return projectConfig[DBTProject.TARGET_PATH_VAR] as string;
-    }
-    return "target";
+    const targetPathOverride = workspace
+      .getConfiguration("dbt")
+      .get<string>("targetPathOverride");
+
+    return (
+      (targetPathOverride
+        ? substituteSettingsVariables(targetPathOverride)
+        : false) ||
+      process.env.DBT_TARGET_PATH ||
+      (projectConfig[DBTProject.TARGET_PATH_VAR]
+        ? (projectConfig[DBTProject.TARGET_PATH_VAR] as string)
+        : false) ||
+      "target"
+    );
```

Comment:
PTAL at `PythonEnvironment` class and ensure that the env var parsing holds true to the rules set in https://github.com/innoverio/vscode-dbt-power-user#environment-variables.

Reply:
Thanks -- that makes a little more sense. I got it working with [this commit](https://github.com/innoverio/vscode-dbt-power-user/pull/521/commits/b429fccb27f0b218558ab29a84e910f0658da4de).

Should [this line](https://github.com/innoverio/vscode-dbt-power-user/blob/master/src/manifest/dbtProject.ts#L120)
also use the `PythonEnvironment` instead of `process.env`?

Reply:
Yeah. good spot!